apps/studio/src/utils/sitemap.ts

```diff
@@ -110,3 +115,46 @@ export const getSitemapTree = (
     children: getSitemapTreeFromArray(resources, null, "/"),
   }
 }
+
+const NUMBER_OF_CARDS_IN_COLLECTION_BLOCK = 3
+export const overwriteCollectionChildrenForCollectionBlock = (
+  sitemap: IsomerSitemap,
+): IsomerSitemap => {
+  // If the current node is a collection, overwrite its children and return
+  if (sitemap.layout === ISOMER_USABLE_PAGE_LAYOUTS.Collection) {
+    return {
+      ...sitemap,
+      children: Array.from({ length: NUMBER_OF_CARDS_IN_COLLECTION_BLOCK }).map(
+        (_, idx) => ({
+          id: `collection-card-${idx}`,
+          title: "Article title",
+          summary: "Article summary",
+          permalink: "/",
+          layout: ISOMER_USABLE_PAGE_LAYOUTS.Article,
+          lastModified: new Date().toISOString(),
+          category: "Category of article",
```

Comment:
[question] don't have to do now, but wondering if we should use the children's title/summary if any